lib/rules/no-implicit-this.js

```diff
@@ -53,7 +53,7 @@ const ARGLESS_DEFAULT_BLUEPRINT = [
 
 export default class NoImplicitThis extends Rule {
   parseConfig(config) {
-    if (config === false || config === undefined || !this.isStrictMode) {
```

Comment:
Evidence that `no-implicit-this` was also wrong: it should skip _if_ the template is strict mode. It just worked accidentally because `isStrictMode` was also completely wrong! 🤪